controller.md

```diff
@@ -0,0 +1,35 @@
+# Controllers
+
+Up to 4 controllers can be used on the switch, they'll match the controller configuration done in the switch parameters.
+
+Controller names is the only "link" we have to tell godot what is the controller switch style. So there is a little bit of semantic in it for the one that need it. It goes as follow: `switch-pad-<id>::<style>::#color`.
+
+For example firt controller is joy-con in dual node vertical it will be `switch-pad-0::dual::#ffe6b90a::#ff283cff` (yes there is 2 colors because left and righ con have both there color). Another example second controller is a left joy-con in horizontal mode ti will be `switch-pad-0::solo-left::#ffe6b90a` and so on.
+
+**Vertical setup** `dual`/`pro`/`handheld` is supported with a pretty generic mapping, nothing special. (this mode also provide left anr right sticks analog axis).
+
+**Horizontal setup** `solo-left`/`solo-right` is also supported with funkier mapping. To provide consistency) the vertical style provide re-oriented D-PAD + 4 buttons + L/R. (No stick axis for this configuration --> fake D-PAD).
```

Comment:
Not really? I don't understand why you would assume there's a D-Pad on the controller in the first place.
One of the supported configurations is the Palma controller (the Pokéball Plus) which has no directional pad, only one button, and an accelerometer (which I previously assumed was a gyroscope).
These kinds of restrictions should be accounted for, because giving yourself restrictions can be a very fun way of doing game development.

Reply:
Ooook, not sure of your point but no worries will add the analog input and make sure the axis makes sense ;) from their dev can use what ever they want engine side.